data/json/items/armor/boots.json

```
@@ -1264,6 +1264,25 @@
     "material_thickness": 1,
     "flags": [ "VARSIZE", "SKINTIGHT" ]
   },
+  {
+    "id": "xlsocks",
+    "type": "ARMOR",
+    "name": { "str_sp": "pair of xl socks" },
```

The other XL items have theirs capitalized
```suggestion
    "name": { "str": "pair of XL socks", "str_pl": "pairs of XL socks" },
```